src/MEDS_tabular_automl/scripts/tabularize_time_series.py

```
@@ -109,6 +119,7 @@ def compute_fn(shard_df):
                 sparse_matrix,
                 window_size,
                 agg,
+                label_df,
```

Ruff (0.8.2)

Function definition does not bind loop variable `label_df`

(B023)